extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -502,6 +553,13 @@ export class MigrationCutoverDialog {
 				}
 			});
 
+			if (tableData.length === 0) {
```

Comment:
This component should only be displayed when the data is expected to be there eventually but isn't.  
I notice the method above that is loading the data checks this._shouldDisplayBackupFileTable() before adding and showing data in the table... this is because the other scenarios won't have the data or expect it.